Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -96,335 +97,3 @@ def dupNamespace : Linter where run := withSetOptionIn fun stx ↦ do
 initialize addLinter dupNamespace
 
 end DupNamespaceLinter
```

Comment:
Can you make sure that there are no "open" namespaces here?

Reply:
Good catch, fixed!